specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -103,6 +103,39 @@
         }
       }
     },
+    "/providers/Microsoft.ResourceHealth/emergingIssues": {
```

Comment:
Its recommended to mark collection GETs as [x-ms-pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable) in case the volume of entities you are returning increases in the future and you need to start paging the response.

Reply:
OK. Sounds good. I'll make that change.

Reply:
It is a collection.  If its a singleton you can leave x-ms-pageable off

Reply:
OK. I'll change to collection with default as suggested. Thanks for the comment.